components/expenses/ExpenseSummary.js

```diff
@@ -70,17 +80,38 @@ const ExpenseSummary = ({
         <H4 mb={2} mr={2} fontWeight="500">
           {isLoading ? <LoadingPlaceholder height={32} minWidth={250} /> : expense.description}
         </H4>
-        {expense?.status && (
-          <ExpenseStatusTag
-            display="block"
-            status={expense.status}
-            letterSpacing="0.8px"
-            fontWeight="600"
-            fontSize="10px"
-            showTaxFormTag={includes(expense.requiredLegalDocuments, 'US_TAX_FORM')}
-            showTaxFormMsg={expense.payee.isAdmin}
-          />
-        )}
+        <Box display="flex" mb={3} width={1} justifyContent={['space-between', 'end']} alignItems="center">
+          {expense?.status && (
+            <Box>
+              <ExpenseStatusTag
+                display="block"
+                status={expense.status}
+                letterSpacing="0.8px"
+                fontWeight="600"
+                fontSize="10px"
+                showTaxFormTag={includes(expense.requiredLegalDocuments, 'US_TAX_FORM')}
+                showTaxFormMsg={expense.payee.isAdmin}
+              />
+            </Box>
+          )}
+          {!isLoading && (
+            <Box display={['flex', 'none']}>
+              <ExpenseAdminActions
+                collective={collective}
+                expense={expense}
+                permissions={pick(expense.permissions, ['canSeeInvoiceInfo', 'canDelete'])}
+                buttonProps={{ size: 32, m: 1 }}
+                linkAction="link"
+                onDelete={() => {
+                  onClose();
+                  if (onDelete) {
+                    onDelete(expense);
+                  }
```

Comment:
If `onClose` prop type is not required, we should also rest if it exists before calling it.
There's a shorthand to do that if you're interested:
```suggestion
                  onClose?.();
                  onDelete?.(expense);
```